release_notes/gitops-release-notes.adoc

```diff
@@ -7,6 +7,13 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+[NOTE]
+====
+For additional information about the OpenShift GitOps life cycle and supported platforms, refer to the link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] and link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShift Container Platform Life Cycle Policy].
```

Comment:
```suggestion
For additional information about the OpenShift {gitops-shortname} life cycle and supported platforms, refer to the link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] and link:https://access.redhat.com/support/policy/updates/openshift[Red{nbsp}Hat {OCP} Life Cycle Policy].
```

Reply:
@Dhruv-Soni11 In the go-compatibility-and-support-matrix.adoc file, please add this information under the IMPORTANT admonition  as a second bullet:

The maintenance support for {OCP} 4.12 on {ibmpowerProductName}, {ibmzProductName}, and {linuxoneProductName} has ended from 17 July 2024. If you are using {gitops-title} on {OCP} 4.12, upgrade to {OCP} 4.13 or later.

Reply:
Fixed.